automatic/graphviz/tools/chocolateyUninstall.ps1

```diff
@@ -12,7 +12,22 @@ if ($key.Count -eq 1) {
             fileType               = 'EXE'
             validExitCodes         = @(0)
             file                   = "$($_.UninstallString.Replace(' /x86=0', ''))"   #"C:\Program Files\OpenSSH\uninstall.exe" /x86=0
+            softwareName           = 'Graphviz*'
         }
+
+        $installLocation = Get-AppInstallLocation $packageArgs.softwareName
```

Comment:
At this point, you should not need to use the `Get-AppInstallLocation` helper, you can query the registry directly by using the `$key` variable (if possible).

Reply:
Fixed. Thank you for the feedback!